### ISSUE TITLE
KMyMoney - Error dispplayed when attempted to save a session

### ISSUE THREAD

**Issue by WetGeek** [getsolus/packages#1449]
### Summary

While testing Discover, I used it to install KMyMoney.  I exercised that application, creating a session for my Brokerage account.  When I'd entered account information and provided some setting information, the last step before quitting the program would be to save the storage for that session, by default in my /home directory.

### Steps to reproduce

1. On a Plasma VM, use Discover to install KMyMoney.
2. Create user information as requested by the application.
3. Create account data for a financial account.  (I used my Brokerage account as an example.)
4. Attempt to save that session and quit the application.

### Expected result

I expect that the storage used by that session would be saved in a file, by default in my /home directory.  I 

### Actual result

![image](https://github.com/getsolus/packages/assets/6805994/3dce50f5-f412-4de7-9096-39ee67e4ab6f)

### Environment

- [X] Is system up to date?

### Repo

Unstable

### Desktop Environment

Plasma

### System details

![image](https://github.com/getsolus/packages/assets/6805994/25715555-e545-415c-8220-8b34e7a05576)

**Comment by TraceyC77** [comment]
I have had KMyMoney installed for a few years. I attempted to reproduce by opening a new KMyMoney file, on my regular Plasma install. I was able reproduce the error, but only if I didn't follow the pop up dialogs as they appeared. And only the first time I did that.

## Process
After entering my personal and some account information, the dialog "Save storage as" appeared, asking me what storage type to use for my data. 
There are two options - XML and SQL.
I chose XML.
The Save As dialog appeared.

### Interrupted workflow - error produced
I canceled out of this. When I then tried to save, I saw the error.

### Normal workflow - saved as normal
 I provided a file name and was able to save the file to my home directory.

Every further attempt produced no error, even if I canceled out of the dialog.

I'll test again on a fresh Plasma install in a VM, and install from Discover.

@WetGeek , did you cancel out of the wizard at any point, before saving the file?
Can you walk me through exactly what you did at each screen, starting from when you enter the user information?

**Comment by TraceyC77** [comment]
@WetGeek , are you able to re-test this now that we've updated to Plasma 6?

**Comment by WetGeek** [comment]
Since it's been a while, and my VM that was set up for testing Discover no longer existed, I cloned a new up-to-date Solus Plasma VM, and installed Discover.  Then I used it to install MyMoney, and repeated my original test to the best of my recollection.

When the new data was input, and the new account created, I saved it, as I had attempted to before:
![image](https://github.com/getsolus/packages/assets/6805994/719a9fcc-9ba0-473d-9fe1-3dfb01989a9a)

This time no error appeared, and although I'm not an experienced MyMoney user, I'm convinced that the error I originally reported no longer exists.  Thus I'll close this issue for now.
